.github/workflows/ci.yml

```diff
@@ -33,3 +33,6 @@ jobs:
 
       - name: Check commits messages
         uses: wagoid/commitlint-github-action@v5
+
+        - name: Run the tests
```

Comment:
There is some indentation fault in yaml due to spaces. Please resolve to run this action.